notebooks/explainer_examples.ipynb

```diff
@@ -4,14 +4,14 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Example Model Servers with Seldon"
+    "# Example Model Explanations with Seldon"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Prerequistes\n",
+    "## Prerequisites\n",
```

Comment:
This 'Prerequistes' typo also appears in other notebooks from the `examples` folder (such as `examples/ambassador/canary/ambassador_canary.ipynb`) as well as in others from the `notebooks` folder (such as `notebooks/istio_example`)

Reply:
We should `sed` the typo across all files.